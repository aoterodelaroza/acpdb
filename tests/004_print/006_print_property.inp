## check: 006_print_property.out -a1e-10
## labels: regression quick

verbose
connect ../zz_source/bsip2-small/bsip2.db
print property
