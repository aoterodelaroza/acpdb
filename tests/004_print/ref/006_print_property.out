%% verbose
%% connect ../zz_source/bsip2-small/bsip2.db
* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file ../zz_source/bsip2-small/bsip2.db
Connected database is sane

%% print property
* PRINT: print the contents of the database 

| id| key| property_type| setid| orderid| nstructures|
| 1| s22.nh3_nh3| 1| 1| 1| 3|
| 2| s22.h2o_h2o| 1| 1| 2| 3|
| 3| s22.h2co2_h2co2| 1| 1| 3| 3|
| 4| s22.formamide_formamide| 1| 1| 4| 3|
| 5| s22.uracil_uracil_hb| 1| 1| 5| 3|
| 6| s22.pyridoxine_aminopyridine| 1| 1| 6| 3|
| 7| s22.adenine_thymine_wcc1| 1| 1| 7| 3|
| 8| s22.ch4_ch4| 1| 1| 8| 3|
| 9| s22.c2h4_c2h4| 1| 1| 9| 3|
| 10| s22.c6h6_ch4| 1| 1| 10| 3|
| 11| s22.c6h6_c6h6_pd| 1| 1| 11| 3|
| 12| s22.pyrazine_pyrazine| 1| 1| 12| 3|
| 13| s22.uracil_uracil_stack| 1| 1| 13| 3|
| 14| s22.indole_c6h6_stack| 1| 1| 14| 3|
| 15| s22.adenine_thymine_stack| 1| 1| 15| 3|
| 16| s22.c2h4_c2h2| 1| 1| 16| 3|
| 17| s22.c6h6_h2o| 1| 1| 17| 3|
| 18| s22.c6h6_nh3| 1| 1| 18| 3|
| 19| s22.c6h6_hcn| 1| 1| 19| 3|
| 20| s22.c6h6_c6h6_t| 1| 1| 20| 3|
| 21| s22.indole_c6h6_t| 1| 1| 21| 3|
| 22| s22.phenol_phenol| 1| 1| 22| 3|
| 23| strange| 3| 1| 1| 1|

